contracts/staking/NativeTokenStakingManager.sol

```diff
@@ -85,6 +90,6 @@ contract NativeTokenStakingManager is
 
     // solhint-disable-next-line no-empty-blocks
     function _reward(address account, uint256 amount) internal virtual override {
-        // TODO: call the native minter precompile to mint `amount` for `account`
+        NATIVE_MINTER.mintNativeCoin(account, amount);
```

Comment:
in ICTT `NativeTokenRemote` we track total supply minted for native coins, do you think anyone would look for total rewards minted?